ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/backtotop/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ReenaOmana : footer container component policy has not been updated to add Back to top component

Reply:
Footer component policy is updated now